libs/auth-api-lib/src/lib/services/delegations.service.ts

```diff
@@ -156,8 +158,10 @@ export class DelegationsService {
           model: DelegationScope,
           where: {
             [Op.and]: [
-              { validFrom: { [Op.lt]: now } },
-              { validTo: { [Op.or]: [{ [Op.eq]: null }, { [Op.gt]: now }] } },
+              { validFrom: { [Op.lte]: today } },
+              {
+                validTo: { [Op.or]: [{ [Op.eq]: null }, { [Op.gte]: today }] },
```

Comment:
Note that today is **inclusive**.
Would be good to verify other places that this should indeed be inclusive